recipes/querynator/meta.yaml

```diff
@@ -27,8 +27,8 @@ requirements:
     - numpy
     - pandas
     - pip
-    - pysam >=0.20.0
     - pytest >=6.2.4
+    - pyvcf3 >=1.0.3
```

Comment:
Hi, the `host:` section should only contain the dependencies needed to run `pip install`, normally that is only `python` and `pip`. You should also be able to remove the `build` section entirely.

Reply:
Hi @marcelm, thanks for the review! Just applied your suggestions. :)